core/primitives/src/telemetry.rs

```diff
@@ -31,6 +31,14 @@ pub struct TelemetryChainInfo {
     pub latest_block_hash: String,
     pub latest_block_height: BlockHeight,
     pub num_peers: usize,
+    /// The `block_production_tracking_delay` from the client config, represented in seconds.
+    pub block_production_tracking_delay: f64,
+    /// The `min_block_production_delay` from the client config, represented in seconds.
+    pub min_block_production_delay: f64,
+    /// The `max_block_production_delay` from the client config, represented in seconds.
+    pub max_block_production_delay: f64,
+    /// The `max_block_wait_delay` from the client config, represented in seconds.
+    pub max_block_wait_delay: f64,
```

Comment:
Who's using the `Deserialize` impl here? Something outside of nearcore, like indexer? 

If there are users of `Deserialize`, it'll be prudent to stick `#[serde::default]` on top of new fields, so that deserializing telemetry from old neards doesn't fail.

Reply:
The telemetry requests are processed by `near-explorer`, not by nearcore.
`serde:default` isn't technically needed, but let's add it nevertheless.

Reply:
If it isn't needed, I'd rathre we remove `Deserialize` impls here

Reply:
Good point. Removed, and tested that telemetry data can still be ingested.